static/js/activity.js

```diff
@@ -18,10 +18,16 @@ exports.IDLE = "idle";
 // case after a server-initiated reload.
 exports.has_focus = document.hasFocus && document.hasFocus();
 
+// We initialize this to false as when opened in browser this should remain
+// false. It is updated true only by the desktop app using electron_bridge
+// and hence would always remain false for all other platforms.
+exports.desktop_has_focus = false;
+
 // We initialize this to true, to count new page loads, but set it to
 // false in the onload function in reload.js if this was a
 // server-initiated-reload to avoid counting a server-initiated reload
 // as user activity.
+
```

Comment:
any reason for this new line here? Just seems to be not consistent with the space between the comment and code above.

Reply:
Sorry my bad. Added by mistake. Will remove it thanks.